exercises/basiccalcobject.js

```diff
@@ -21,6 +53,34 @@ class Calculator{
 		//otherwise return false (too many numbers stored)
 	}
 	calculate(){
+		switch(this.operator){
+			case "+":
+				const add_result = this.number1 + this.number2;
+				this.number1 = null;
```

Comment:
since you are always setting them to null every time, you might as well do this outside of the switch.  Return outside, too.